JetUtilities/src/crystalBallRes.cc

```diff
@@ -0,0 +1,213 @@
+#include "JetMETAnalysis/JetUtilities/interface/crystalBall.h"
+#include "JetMETAnalysis/JetUtilities/interface/crystalBallRes.h"
+
+#include <TH1.h>
+#include <TF1.h>
+#include <TCanvas.h>
+
+#include <iostream>
+#include <vector>
```

Comment:
Again, it is a style issue. I'm trying to make your style and the preexisting style match up.